tests/test_update_ssh_keys.rs

```diff
@@ -0,0 +1,524 @@
+        _ = add_one_ssh_key(
+            ctx.ssh_dir.to_str().unwrap(), // ssh_dir
+            "valid1",                      // pubkeyname
+            "one",                         // testname
+            false,                         // is_force
+            true,                          // is_replace
+            ["valid1"].to_vec(),           // assert_keys
+        );
+
+        // "update-ssh-keys --add one valid2.pub"
+        _ = add_one_ssh_key(
+            ctx.ssh_dir.to_str().unwrap(), // ssh_dir
+            "valid2",                      // pubkeyname
+            "one",                         // testname
+            false,                         // is_force
+            true,                          // is_replace
+            ["valid2"].to_vec(),           // assert_keys
+        );
+    }
+
+    #[test]
+    fn test_no_replace() {
+        let ctx = setup_tests();
+
+        // "update-ssh-keys --delete one valid2.pub"
+        del_one_ssh_key(ctx.ssh_dir.to_str().unwrap(), "valid2", "one");
+
+        // "update-ssh-keys --add one valid1.pub"
+        _ = add_one_ssh_key(
+            ctx.ssh_dir.to_str().unwrap(), // ssh_dir
+            "valid1",                      // pubkeyname
+            "one",                         // testname
+            false,                         // is_force
+            true,                          // is_replace
+            ["valid1"].to_vec(),           // assert_keys
+        );
+
+        // "update-ssh-keys --no-replace --add one valid2.pub"
+        match add_one_ssh_key(
+            ctx.ssh_dir.to_str().unwrap(), // ssh_dir
+            "valid2",                      // pubkeyname
+            "one",                         // testname
+            false,                         // is_force
+            false,                         // is_replace
+            ["valid1"].to_vec(),           // assert_keys
+        ) {
+            Ok(()) => panic!("unexpected test success"),
+            Err(err) => {
+                // Error would be like "keys with name 'one' already exist".
+                assert!(format!("{}", err).contains("keys with name"));
+                assert!(format!("{}", err).contains("already exist"));
+            }
+        }
+
+        // "update-ssh-keys --no-replace --add-force one valid2.pub"
+        _ = add_one_ssh_key(
+            ctx.ssh_dir.to_str().unwrap(), // ssh_dir
+            "valid2",                      // pubkeyname
+            "one",                         // testname
+            true,                          // is_force
+            false,                         // is_replace
+            ["valid1"].to_vec(),           // assert_keys
+        );
+    }
+
+    #[test]
+    fn test_add_two() {
+        let ctx = setup_tests();
+
+        // "update-ssh-keys --add one valid1.pub"
+        test_add_one_file();
+
+        // "update-ssh-keys --add two valid2.pub"
+        _ = add_one_ssh_key(
```

Comment:
This supresses the error, please handle it

Reply:
Done